src/services/webclient.ts

```diff
@@ -4054,4 +4060,16 @@ export class WebClientService {
             this.batteryStatusTimeout = null;
         }
     }
+
+    /**
+     * Log chat message's metadata for debugging purposes.
+     */
+    private logChatMessages(type: string, subType: string, messages: threema.Message[]) {
+        for (const message of messages) {
+            const idHex = u8aToHex(base64ToU8a(message.id));
+            this.$log.debug('[MessageInspector]', `${type}/${subType}: direction=${message.isOutbox ? 'out' : 'in'}, ` +
+                `id=${idHex}. type=${message.type}, state=${message.state !== undefined ? message.state : '?'}, ` +
```

Comment:
Should this `.` be a `,`?

```suggestion
                `id=${idHex}, type=${message.type}, state=${message.state !== undefined ? message.state : '?'}, ` +
```

Reply:
Yes, good catch!

Reply:
Or, as we say in Swiss German, "Tüpflischisserei" :smile: